train.py

```diff
@@ -16,7 +17,11 @@ def train(model, optimizer, scheduler, loss_function, train_iter, val_iter, args
         for batch in tqdm.tqdm(train_iter):
             optimizer.zero_grad()
             scores = model(batch.src, batch.trg)
-            loss = loss_function(scores[:-1,:,:].view(-1, scores.shape[2]), batch.trg[1:,:].view(-1))
+            if isinstance(loss_function, VonMisesFisherLoss):
```

Comment:
@artidoro I had forgotten earlier why I had this conditional here. It is because the loss function needs to be passed different values with cross-entropy vs vMF loss. This is probably the neatest way to do things since the loss function won't have access to the embeddings otherwise.